### PR TITLE
Some small changes

### PR DESCRIPTION
**Pull request by loganhasson**
cc @matbalez

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,43 +10,43 @@ In this lesson, we'll walk through the steps you need to take to solve a lab on
...
+1. The first, and simplest, is to just click the `Open` button in the toolbar at the top of a lab on Learn—don't try to do this just yet, you'll do this in the next lesson. When you click Open, you'll notice a Terminal window briefly pop up, then your text editor will open with the contents of the lab's directory. You're now ready to start writing code to solve the lab.
+2. Alternatively, from a Terminal window, you can type the `learn open` command which will open up your current lab
 
 You'll notice that on Learn, the light labelled "Fork" will turn green after you've opened up a lab.
 
-So what's going on here? Behind the scenes, the Learn CLI is (1) forking this lab's repository on Github (2) running `git clone` to clone the contents down to your computer (3) running `cd` to change directory into the lab's directory then (4) opening your text editor so you can get to work. 
+So what's going on here? Behind the scenes, the Learn CLI is (1) forking this lab's repository on Github (2) running `git clone` to clone the contents down to your computer (3) running `cd` to change directory into the lab's directory then (4) opening your text editor so you can get to work.
```
Comment (loganhasson) [inline README.md:18]: are we worried about users not having a clue what 'forking' is, etc?

File: README.md [hunk 2]
```diff
@@ -10,43 +10,43 @@ In this lesson, we'll walk through the steps you need to take to solve a lab on
...
 
 In order to submit your solution to Learn, just run `learn submit`. That's all it takes.
 
-You'll notice that on Learn, the light labelled "Pull Request" will turn green when your code has been submitted. 
+You'll notice that on Learn, the light labelled "Pull Request" will turn green when your code has been submitted.
 
-So what's going on here? Behind the scenes, the Learn CLI is (1) using the `git push` command to push your coded solution to Github and (2) opening up a Pull Request on the original (un-forked) repository. The cool thing about this is that in the process of solving labs on Learn, you're building up your Github profile!
+So what's going on here? Behind the scenes, the Learn CLI is (1) using the `git add` and `git commit` commands to commit your changes then (2) using the `git push` command to push your coded solution to GitHub and (3) opening up a Pull Request on the original (un-forked) repository. The cool thing about this is that in the process of solving labs on Learn, you're building up your GitHub profile!
```
Comment (loganhasson) [inline README.md:41]: similar question to above

File: README.md (hunk 1 above)
Comment (matbalez-zz) [inline README.md:18]: i am assuming that there is a short lesson ahead of this one that introduces some of the basic git stuff. we'll need to read it in context to know for sure.

Comment (loganhasson) [inline README.md:18]: sweet

**Pull request merged**